src/controllers/routes.ts

```diff
@@ -10,20 +10,32 @@ import {
     getGroups,
     getUserById,
     getUsers,
+    login,
     updateGroup,
     updateUser,
+    verifyToken,
 } from './controllers';
 import { validateSchema } from '../validation/validation';
 import { createUserSchema, updateUserSchema } from '../validation/schema';
 
 export const routes = (router: Router) => {
-    router.route('/users').get(getUsers).post(validateSchema(createUserSchema), createUser);
+    router.route('/users').get(verifyToken, getUsers).post(verifyToken, validateSchema(createUserSchema), createUser);
 
-    router.route('/users/:uuid').get(getUserById).put(validateSchema(updateUserSchema), updateUser).delete(deleteUser);
+    router
+        .route('/users/:uuid')
+        .get(verifyToken, getUserById)
+        .put(verifyToken, validateSchema(updateUserSchema), updateUser)
+        .delete(deleteUser);
 
-    router.route('/groups').get(getGroups).post(createGroup);
+    router.route('/groups').get(verifyToken, getGroups).post(verifyToken, createGroup);
```

Comment:
Makes sense to use https://expressjs.com/en/api.html#router.use to avoid specifying the `verifyToken` middleware for each route. Thus we also avoiding risk of forgetting to add the middleware to some routes

Reply:
Fixed, thanks!